src/routes/mainRouter.js

```diff
@@ -1,7 +1,14 @@
-const express = require('express');
+const express = require("express");
+const { signUp, logIn, logOut } = require("../controllers/userController");
+const signupDataValidate = require("../middlewares/signupDataValidate");
+const loginDataValidate = require("../middlewares/ loginDataValidate");
+const authenticateUser = require("../middlewares/authenticateUser");
 const router = express.Router();
-const mainController = require('../controllers/mainController.js');
 
-router.get('/', mainController.get);
-  
-module.exports = router;
\ No newline at end of file
+//user
+router.post("/signup", signupDataValidate, signUp);
+router.post("/login", loginDataValidate, logIn);
+router.get("/getuser", authenticateUser);
```

Comment:
Why did you decide to have separate login and getuser endpoints? It looks like the `/login` endpoint only does validation of the user fields. I'd suggest combining these so that there's just one authentication endpoint that also does validation.

Reply:
This part depends really  it of the front-end team .

Reply:
Good point! Would you be able to check in with them? Do y'all normally have these types of conversations in slack?